### PR TITLE
Fix function logic description in BiPoolManager.sol

### PR DESCRIPTION
**Pull request by tobikuhlmann**
### Description

fixes wrong function logic description

### Other changes

no

### Tested

forge tests pass

### Related issues

- Fixes https://github.com/mento-protocol/mento-general/issues/154

### Backwards compatibility

fully just comment change

### Documentation

no docs

### CODE REVIEW

**Review by philbow61 — APPROVED** [review]

File: contracts/StableTokenRegistry.sol [hunk 1]
```diff
@@ -79,7 +79,7 @@ contract StableTokenRegistry is Initializable, Ownable {
   }
 
   /**
-   * @notice Removes unwamted token instances.
+   * @notice Removes unwanted token instances.
```
Comment (ninocomputer) [inline contracts/StableTokenRegistry.sol:82]: let's revert this change

**Review by ninocomputer — CHANGES_REQUESTED** [review]

File: contracts/StableTokenRegistry.sol (hunk 1 above)
Comment (tobikuhlmann) [reply contracts/StableTokenRegistry.sol:82]: I want to leave it in, it does no hurt, and at least it shows as an addressed finding in the final audit report instead of unaddressed as well as correct language in the git history

Comment (tobikuhlmann) [reply contracts/StableTokenRegistry.sol:82]: since you closed the other PR we can just do it here

Comment (ninocomputer) [reply contracts/StableTokenRegistry.sol:82]: `StableTokenRegistry.sol` is removed from main branch and has never been deployed on mainnet. Also lets only address things mentioned in the ticket, in order to keep it contained and straightforward(only because of the audit, otherwise we could do it as a drive-by change)

Comment (tobikuhlmann) [reply contracts/StableTokenRegistry.sol]: ok. Why then is the registry in the develop branch here?

Comment (ninocomputer) [reply contracts/StableTokenRegistry.sol]: Develop branch used to be McMint, so any changes not related to McMint were merged into the main back in the day

**Review by ninocomputer — APPROVED** [review]
